src/rpc/CMakeLists.txt

```diff
@@ -83,6 +87,8 @@ set(daemon_rpc_server_private_headers
 monero_private_headers(rpc
   ${rpc_private_headers})
 
+set(rpc_pub_private_headers)
```

Comment:
If it is empty, why we need it here?

Reply:
This is the style in this file. The empty set is declared like this and passed to the `monero_add_library` function. I can revert to `""` is desired, I only did it this way to follow the existing style.